website/src/components/CustomerTestimonials/index.tsx

```diff
@@ -102,59 +114,76 @@ const TestimonialBox = ({
 };
 
 export default function CustomerTestimonials() {
-  const scrollRef = useRef<HTMLDivElement>(null);
+  // const scrollRef = useRef<HTMLDivElement>(null);
 
-  const scrollLeft = () => {
-    if (scrollRef.current) {
-      scrollRef.current.scrollBy({
-        left: -540,
-        behavior: "smooth",
-      });
-    }
-  };
+  // const scrollLeft = () => {
+  //   if (scrollRef.current) {
+  //     scrollRef.current.scrollBy({
+  //       left: -540,
+  //       behavior: "smooth",
+  //     });
+  //   }
+  // };
 
-  const scrollRight = () => {
-    if (scrollRef.current) {
-      scrollRef.current.scrollBy({
-        left: 540,
-        behavior: "smooth",
-      });
-    }
-  };
+  // const scrollRight = () => {
+  //   if (scrollRef.current) {
+  //     scrollRef.current.scrollBy({
+  //       left: 540,
+  //       behavior: "smooth",
+  //     });
+  //   }
+  // };
 
   return (
     <section className="bg-neutral-950 py-24">
-      <div className="relative mx-auto max-w-screen-lg">
-        <div className="px-8 sm:px-16 md:px-24 mb-12 md:mb-16">
+      <div className="relative flex flex-col lg:flex-row items-start lg:items-center justify-center max-w-screen-2xl">
+        <div className="px-4 sm:px-16 lg:px-10 xl:px-0 xl:pr-16 mb-12 md:mb-16">
           <div>
             <h3
-              className={`text-white text-3xl leading-5 md:text-4xl lg:text-5xl tracking-tight font-medium inline-block text-left mb-2 ${manrope.className}`}
+              className={`text-white text-3xl md:text-4xl leading-5 xl:text-5xl tracking-tight font-medium inline-block text-wrap sm:text-nowrap text-left ${manrope.className}`}
             >
               Customers{" "}
-              <FaHeart className="text-red-500 w-12 h-12 mx-1 inline-block" />{" "}
+              <FaHeart className="text-primary-450 w-12 h-12 mx-1 inline-block" />{" "}
               us,
             </h3>
-            <p className="text-neutral-500 text-2xl md:text-3xl font-medium">
+            <h3 className="text-neutral-600 text-3xl md:text-4xl leading-12 xl:text-5xl text-wrap sm:text-nowrap tracking-tight font-medium mb-6">
               and we love them back.
-            </p>
+            </h3>
+            <ActionLink href="/contact/sales" color="white">
```

Comment:
```suggestion
            <ActionLink href="/contact/sales" color="white">
```

Just double-checking - is `white` the color we're using for other light-on-dark ActionLinks?

Reply:
yes so that when you hover it turns orange

Reply:
Hm, I don't think that's right. Here's the ActionLink from the hero

```
              <ActionLink
                size="lg"
                href="https://app.firezone.dev/sign_up"
                color="neutral-100"
                transitionColor="primary-450"
              >
                Get started for free
              </ActionLink>
```

Reply:
@Patticatti  ^^ 🤷 ??

Reply:
huh did i not fix it? I changed it to neutral-100

Reply:
now i fixed it - thoguht i pushed but i didnt

Reply:
Could you make sure the ActionLinks are consistent? I.e. I think you need these properties on this one:

```
              <ActionLink
                size="lg"
                href="https://app.firezone.dev/sign_up"
                color="neutral-100"
                transitionColor="primary-450"
              >
```